ARE/middleware/src/main/java/eu/asterics/mw/are/AsapiSupport.java

```diff
@@ -1557,15 +1557,13 @@ private void storeData(String data, String filepath, RES_TYPE resourceType, Char
     }
 
     /**
-     * stores data with UTF-8
+     * stores data with UTF-8 to folder ARE/data
      *
      * @param data
-     * @param path
-     * @param filename
+     * @param filepath path + filename of the data to store (or only filename if no subpath is needed)
      * @throws AREAsapiException
      */
-    public void storeData(String data, String path, String filename) throws AREAsapiException {
-        String filepath = path + "/" + filename;
+    public void storeData(String data, String filepath) throws AREAsapiException {
         this.storeData(data, filepath, RES_TYPE.DATA, Charsets.UTF_8);
```

Comment:
so only the UTF-8 version will be supported?
what about the version with the given charset name?

Reply:
I think both `UTF8` and `UTF16` can represent all possible UTF-characters, so `UTF16` ist not better to "support international characters" (see e.g. https://webmasters.stackexchange.com/questions/48618/how-utf-8-represents-more-than-256-characters-and-when-to-use-utf-16 ). But yes, we maybe should agree on one consistent encoding, so if the major encoding of AsTeRICS is `UTF16`, I'll change this method to use `UTF16`.

Reply:
i changed the method so that all data (models or data stored to `ARE/data`) is encoded with UTF16. This was the only feedback, so I'll merge this now.